stored_requests/backends/db_provider/mysql_dbprovider.go

```diff
@@ -75,7 +84,65 @@ func (provider *MySqlDbProvider) ConnString() string {
 		buffer.WriteString(provider.cfg.Database)
 	}
 
-	return buffer.String()
+	// TSL connection
+	if provider.cfg.ServerCa != "" && provider.cfg.ClientCert != "" && provider.cfg.ClientKey != "" {
```

Comment:
If we're expecting either none or all three of these to be set, we could add some validation at startup that warns if only one or two of these are set.